pkg/controller/generate-registry/generate.go

```diff
@@ -285,3 +287,45 @@ func checkSLSAProvenance(assetName, tagName string) *registry.SLSAProvenance {
 		Asset: &assetInfo.Template,
 	}
 }
+
+func checkChecksumCosign(pkgInfo *registry.PackageInfo, checksumAssetName string, assetNames map[string]struct{}) *registry.Cosign {
+	var signatureAssetName string
+	for _, suf := range []string{"-keyless.sig", ".sig"} {
+		if _, ok := assetNames[checksumAssetName+suf]; ok {
+			signatureAssetName = checksumAssetName + suf
+			break
+		}
+	}
+	if signatureAssetName == "" {
+		return nil
+	}
+	var certificateAssetName string
+	for _, suf := range []string{"-keyless.pem", ".pem"} {
+		if _, ok := assetNames[checksumAssetName+suf]; ok {
+			certificateAssetName = checksumAssetName + suf
+			break
+		}
+	}
+	if certificateAssetName == "" {
+		return nil
+	}
+
+	downloadURL := fmt.Sprintf("https://github.com/%s/%s/releases/download/{{.Version}}/",
+		pkgInfo.RepoOwner, pkgInfo.RepoName)
+	return &registry.Cosign{
+		Opts: []string{
+			"--certificate-identity-regexp",
+			fmt.Sprintf(
+				`^https://github\.com/%s/%s/\.github/workflows/.+\.ya?ml@refs/tags/`,
+				regexp.QuoteMeta(pkgInfo.RepoOwner),
+				regexp.QuoteMeta(pkgInfo.RepoName),
```

Comment:
```suggestion
				pkgInfo.RepoOwner,
				pkgInfo.RepoName,
```

In my understanding, regular expression metacharacters aren't available in GitHub repository owners and names.
So we don't need to use [regexp.QuoteMeta](https://pkg.go.dev/regexp#QuoteMeta) here.
But there is no problem even if it is used.